### ISSUE TITLE
Fix issue with LR

### ISSUE THREAD

**Issue by nestorcarmona** [julien-blanchon/ProjetFixMatch#6]
the formula for the learning rate: 
```lr_lambda = lambda step: LR * torch.cos(torch.tensor((7 * torch.pi * (step)) / (16 * EPOCHS)))```

should start at LR for step = 0.

However, it is not the case, it starts at 0.0009

hence, i multiplied by  * 100 / 3 in order to get the right LR_0

Find why and fix it
